rocAL/rocAL_pybind/run.sh

```diff
@@ -50,5 +50,5 @@ else
   do
     echo "Going to install $WHEEL_NAME"
   done
-  pip$PYTHON_VERSION install $WHEEL_NAME
+  pip3 install $WHEEL_NAME
```

Comment:
@rrawther did you mean to check this in? In this PR?

Reply:
@rrawther The test shell script passes otherwise. 
@kiritigowda Can merge once above comment is resolved

Reply:
reverting it